docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -32,6 +32,9 @@ The following topics describe how to configure each supported output:
 * <<file-output>>
 * <<console-output>>
 
+If you've secured the {stack}, also read <<securing-{beatname_lc}>> for more about
```

Comment:
You probably need an additional check wheter or not `redis-output` is defined, as we have not tested nor documented it in the `apm-server.yml`.